### ISSUE TITLE
a bug in `div` with a negative dividend

### ISSUE THREAD

**Issue by Yaffle** [vukicevic/crunch#24]
```
var crunch = new Crunch();
var a = crunch.parse("-10");
var b = crunch.parse("10");
var c = crunch.div(a, b);
console.log(c); // [-0], should be [-1]
```
P.S.
https://github.com/vukicevic/crunch/blob/master/crunch.js#L417 - `cmp(u, k)` should compare absolute values, may be

**Comment by vukicevic** [comment]
Fixed in #29

**Comment by Yaffle** [comment]
Thanks!
